source/2020-09-25-the-ember-times-issue-164.md

```diff
@@ -25,14 +26,11 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [EmberJS with REST API 💪](https://github.com/ember-learn/ember-blog/pull/768)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[John Costanzo (@jrock2004)](https://twitter.com/jrock2004) wrote a blog titled [EmberJS with REST API](https://github.com/ember-learn/ember-blog/pull/768). The blog details about how to get your APIs to work well with Ember Data out of the box if it follows REST API conventions. If in case your APIs don't follow the REST conventions, the post details about how to modify your Ember App' serializer to make it work with Ember data. 
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The blog also details about using `keyForAttribute` to make some of the manual work a little easier, so you do not have to map each API properties to the ember model properties. Head over to [the blog post](https://github.com/ember-learn/ember-blog/pull/768) today and give it a read.
```

Comment:
```suggestion
The blog also discusses using [`keyForAttribute`](https://api.emberjs.com/ember-data/release/classes/JSONAPISerializer/methods/keyForAttribute?anchor=keyForAttribute) to make some of the manual work a little easier, so that you do not have to map each API property to the Ember model property. Head over to [the blog post](https://jcwebconcepts.net/emberjs-with-rest-api/) today and give it a read.
```